### ISSUE TITLE
vue/singleline-html-element-content-newline breaks <router-link> visual display

### ISSUE THREAD

**Issue by mariusa** [vuejs/eslint-plugin-vue#1749]
Found similar issue: https://github.com/vuejs/eslint-plugin-vue/issues/1633

**What rule do you want to change?**
vue/singleline-html-element-content-newline

**Does this change cause the rule to produce more or fewer warnings?**
don't know

**How will the change be implemented? (New option, new default behavior, etc.)?**
don't know

**Please provide some example code that this change will affect:**

```vue
<router-link :to="parent">test</router-link>
```

**What does the rule currently do for this code?**
<router-link :to="parent">
                    test                
</router-link>

It adds whitespace, which leads to rendering the link in browser as "test_" (extra underlining until closing the tag)

**What will the rule do after it's changed?**
stop adding whitespace:
<router-link :to="parent">test</router-link>

**Additional context**

**Comment by FloEdelmann** [comment]
If `<a>` is ignored, `<router-link>` and `<nuxt-link>` should be ignored as well.

**Comment by mariusa** [comment]
`<a href="">test</a>` works fine (it is left like this), just tested.  `<router-link>` is changed to add whitespace.
